### PR TITLE
PC-574 Updating copy text to reflect January 2024 changes

### PR DESCRIPTION
**Pull request by Glenn-Clarke**
- Updated copy text to reflect January 2024 estimates for cost savings.

Text in recommendations summary page:
image

Text beneath each recommendation:
image

### CODE REVIEW

File: SeaPublicWebsite/Views/EnergyEfficiency/YourRecommendations.cshtml [hunk 1]
```diff
@@ -42,9 +42,13 @@
...
-                The costs and savings we provide are estimates and may not be the same in practice. 
-                The savings displayed are based on the October 2022 energy price cap. 
-                As energy prices change, so will the estimates of savings.
+                The costs and savings we provide are estimates
+                and may not be the same in practice. The savings
+                displayed are based on the January 2024 energy
+                price cap for gas and electricity bills, and on the
+                RdSAP energy prices for all other heating fuel
+                types. As energy prices change, so will the
+                estimates of savings.
```
Comment (jamiehumphries) [inline SeaPublicWebsite/Views/EnergyEfficiency/YourRecommendations.cshtml:51]: I'd prefer keeping the style of the original text here and having one sentence per line.

It makes the breaks more readable and the changes in the commit more obvious.

Typically, I break paragraphs in code after each full stop. If that makes a line seem too long, then look for an appropriate place to break the sentence, like a comma or an "and".

In this case, that would be:

```
The costs and savings we provide are estimates and may not be the same in practice.
The savings displayed are based on the January 2024 energy price cap for gas and electricity bills,
and on the RdSAP energy prices for all other heating fuel types.
As energy prices change, so will the estimates of savings.
```

File: SeaPublicWebsite/Views/Shared/_Recommendation.cshtml [hunk 2]
```diff
@@ -24,10 +24,14 @@
...
-                  "The savings displayed are based on the October 2022 energy price cap. " +
-                  "As energy prices change, so will the estimates of savings."
+           Text = "These figures are for a typical installation in a home " +
+                  "like yours. The costs and savings you experience will " +
+                  "be affected by your circumstances, the details of the " +
+                  "installations and energy prices. The savings displayed " +
+                  "are based on the January 2024 energy price cap for gas " +
+                  "and electricity bills, and on the RdSAP energy prices " +
+                  "for all other heating fuel types. As energy prices change, " +
+                  "so will the estimates of savings."
```
Comment (jamiehumphries) [inline SeaPublicWebsite/Views/Shared/_Recommendation.cshtml:34]: Similarly here, I'd prefer we broke the text up at more considered places, e.g.
```
Text = "These figures are for a typical installation in a home like yours. " +
      "The costs and savings you experience will be affected by your circumstances, " +
      "the details of the installations and energy prices. " +
      "The savings displayed are based on the January 2024 energy price cap for gas and electricity bills, " +
      "and on the RdSAP energy prices for all other heating fuel types. " +
      "As energy prices change, so will the estimates of savings."
```

**Review by jamiehumphries — CHANGES_REQUESTED** [review]
Sorry to nitpick, but I'd prefer we laid this text out differently by keeping with the style that was already in the codebase.

Also, your commits should have a short message to them. This one is just called:
```
PC-574
```
but should be something like you have for this pull request:
```
PC-574: Updating copy text to reflect January 2024 changes.
```

The Git history should provide a quick summary at a glance, without the need to cross reference with ticket numbers.

**Comment by Glenn-Clarke** [comment]
> Sorry to nitpick, but I'd prefer we laid this text out differently by keeping with the style that was already in the codebase.
> 
> Also, your commits should have a short message to them. This one is just called:
> 
> ```
> PC-574
> ```
> 
> but should be something like you have for this pull request:
> 
> ```
> PC-574: Updating copy text to reflect January 2024 changes.
> ```
> 
> The Git history should provide a quick summary at a glance, without the need to cross reference with ticket numbers.

As a bad habit I'd been putting the short description on the following line, so I've pulled it up to the top line so it shows now :)

**Review by jamiehumphries — APPROVED** [review]

**Pull request merged**